js/interface.js

```diff
@@ -2371,6 +2371,7 @@ var DynamicLists = (function() {
       if (limit && limit.length && /^\d+$/.test(limit)) {
         data.enabledLimitEntries = true;
         data.limitEntries = parseInt(limit, 10);
+        data.limitEntriesText = constructEntriesText(data);
```

Comment:
I don't like that the copy has gone from a template file, which is updateable through a widget update deployment, to being saved in the widget configuration, which is only updateable if users open and save the widget configuration in Studio.

Reply:
Then it would be better to return this solution to the template file and remove from widget configuration, right?

Reply:
Correct

Reply:
@tonytlwu  Return solution to a template.